README.md

```diff
@@ -13,18 +13,26 @@ The SPIFFE Helper is a simple utility for fetching X.509 SVID certificates from
 
 If `-config` is not specified, the default value `helper.conf` is assumed. 
 
-The flag `-exitWhenReady` is also supported.
+CLI options:
+
+ | Flag name                       | Description                                                         |
+ |---------------------------------|---------------------------------------------------------------------|
+ | `-config`                       | Path to the configuration file                                      |
+ | `-help`                         | Print interactive help                                              |
+ | `-daemon-mode={true|false}`     | Overrides `daemon_mode` in the config file.                         |
```

Comment:
I think you need to escape the pipe. This line is not rendering properly

Reply:
Good catch. Fixed